curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/6552008c0d9d9075cbec9772.md

```diff
@@ -7,17 +7,16 @@ dashedName: step-13
 
 # --description--
 
-The first kind of cipher you are going to build is called a *Caesar* cipher. Specifically, you will take each letter in your message, find its position in the alphabet, take the letter located after 3 positions, and replace the original letter with the new letter.
-
-Start by finding the position of the first letter in the string. One way is to call the `.find()` string method on the string you want to parse:
+You are going to use the `.find()` method to find the position in the alphabet of each letter in your message. A method is similar to a function, but it belongs to an object.
 
 ```py
-text.find('W')
+sentence = 'My brain hurts!'
+sentence.find('r')
 ```
 
-Above, uppercase `'W'` is the character you want to locate inside the string stored in the `text` variable. The method will return `6`, which is the index of the `'W'` character inside the string stored in the `text` variable.
+Above, the `.find()` method is *called on* `sentence` (the string to search in), and `'r'` (the character to locate) is passed as the argument. The `sentence.find('r')` call will return `4`, which is the index of the first occurrence of `'r'` in `sentence`.
 
-At the end of your code, call `.find()` on your `alphabet` string and pass `text[0]` to the method. Note that a method is just a function that belongs to an object (you will learn more about that in another project).
+At the end of your code, call `.find()` on `alphabet` and pass `'z'` as the argument to the method.
```

Comment:
What about printing the result as well to demonstrate what `find()` actually returns?

Reply:
I wasn't thinking about that since the following steps are:
-> `alphabet.find(text[0])` -> `index = alphabet.find(text[0])` -> `print(index)`
Do you think we should do instead
 -> `alphabet.find('z')` -> `index = alphabet.find('z')` -> `print(index)` and only after that change `index` into  `alphabet.find(text[0])`?

Reply:
I was thinking something like this to teach what `find()` would return:
`print(find.alphabet('z'))` -> remove `print()` call -> `index = alphabet.find(text[0])` -> `print(index)`

What find would return is also mentioned in the example of this instruction, so it is fine even if my suggestion isn't implemented.

Reply:
Let's go forward with the current implementation.